app/controllers/application_controller.rb

```diff
@@ -68,6 +68,6 @@ def set_view_paths
     # Add the default templates directory to the top of view_paths
     prepend_view_path 'app/views/shinycms'
     # Apply the configured theme, if any, by adding it above the defaults
-    prepend_view_path Theme.current.view_path if Theme.current
+    prepend_view_path Theme.current.view_path if Theme.current( current_user )
```

Comment:
Should the `.current` call in `Theme.current.view_path` pass in the `current_user` arg too?